pkg/ops/stakeOps.go

```diff
@@ -74,24 +74,24 @@ func Deposit(ctx context.Context, logger log.Logger) error {
 	copy(dat32[:], dat)
 	stakeAmt, err := tmaster.GetUintVar(nil, dat32)
 	if err != nil {
-		return errors.Errorf("fetching stake amount failed: %s", err.Error())
+		return errors.Wrap(err, "fetching stake amount")
 	}
 
 	if balance.Cmp(stakeAmt) < 0 {
-		return errors.Errorf("insufficient balance (%s), mining stake requires %s TRB",
+		return errors.Wrapf(err, "insufficient balance (%s), mining stake requires %s TRB",
```

Comment:
Where is this error from? Please be careful about such cases as with this the error will be masked and not even returned.

Reply:
> Where is this error from? Please be careful about such cases as with this the error will be masked and not even returned.

I will be careful. thanks.